client/my-sites/email/email-management/index.jsx

```diff
@@ -137,6 +137,19 @@ class EmailManagement extends React.Component {
 				secondaryAction: translate( 'Add Email Forwarding' ),
 				secondaryActionURL: emailManagementForwarding( selectedSiteSlug, selectedDomainName ),
 			};
+			if ( isMappedDomain( selectedDomain ) ) {
+				Object.assign( emptyContentProps, {
+					title: translate( 'G Suite is not supported on this domain' ),
+					line: translate(
+						'To enable G Suite on %(domain)s, please have its name servers changed to that of WordPress.com.',
+						{ args: { domain: selectedDomainName } }
+					),
+					action: translate( 'How to change your name servers' ),
+					actionURL:
+						'https://en.support.wordpress.com/domains/map-existing-domain/#change-your-domains-name-servers',
```

Comment:
Let's avoid the `en.` subdomain to better serve non-English users (or maybe use the [`localizeUrl()` function](https://github.com/Automattic/wp-calypso/blob/bd07030f74ff1cfdc9e5929d23811d18bef0c4c0/client/lib/i18n-utils/utils.js#L180)).

Reply:
Thanks -  I missed that